linked_list/implementation.py

```diff
@@ -1,40 +1,91 @@
 from .interface import AbstractLinkedList
-
+from .node import Node
+from functools import reduce
 
 class LinkedList(AbstractLinkedList):
     """
     Implementation of an AbstractLinkedList inteface.
     """
 
-    def __init__(self, elements=None):
-        pass
+    def __init__(self, elements=[]):
+        #self.elements = elements
+        self.start = None
+        self.end = None
+        for elem in elements:
+            self.append(elem)
 
     def __str__(self):
```

Comment:
You can make it a one liner with `return '[' + ', '.join(str(element) for element in self) + ']'`